### DISCUSSION TITLE
[Piano Song] - Runaway, AURORA

### DISCUSSION THREAD

**Discussion by xXMidnight** [Monika-After-Story/MonikaModDev#7813] [general - piano notes]
### Verse 1

r 4

4 r 4 2 3

r 4

4 r 4 2

r 4

4 r 4 2 3

r 4

4 r 4 2

### Verse 2

r 6 6 7 6 6 6 4
I was listening to the ocean

3 r 6 7 6 6 6 7
I saw a face in the sand

4 r 4 3 r 7
But when I picked it up

7 9 9 u 7 6 r 4 3
Then it vanished away from my hands

4 r 3 4 r
Down

3 r 6 7 6 6 6 4
I had a dream, I was seven

3 r 6 7 6 6 6
Climbing my way in a tree

7 r 3 r 7 9
I saw a piece of heaven

9 [ 9 7 6 r 4 r 4 3
Waiting impatient for me

4 r 3 4 r
Down

### Verse 3

4 r 7 6 4 r 4 3
And I was running far away

4 r 7 6 7 9 7 7
Would I run off the world someday?

7 7 9 6
Nobody knows

6 7 6 9 0 7 6 r 4
Nobody knows, and

r 7 6 4 r 7 6
I was dancing in the rain

4 r 7 6 6 7 9 7 u 7
I felt alive and I can't complain

7 [ 9 u 7
But now take me home

9 7 9 u 7 6 7
Take me home where I belong

9 0 7 9 u 7 6 7 u 7 u 7 6
I can't take it anymore

### Verse 4

6 7 7 6 6 6 6
I was painting a picture

r 3 r r 6 7 6 6 6 4
The picture was a painting of you, and

r 4 3 3 r 7 7 9 9
For a moment I thought you were here

7 u 7 6
But then again

4 r 4 3
It wasn't true

4 r 3 4 r
Down

7 7 0 - [ - 0 9 9 6
And all this time I have been lying

7 9
Oh

7 0 - [ - 0 9 9 0
Lying in secret to myself

9 0 6 6 6 7 7 9 9 7 6 6 7 6 7
I've been putting sorrow on the farthest place on my shelf

u 9 7 u 7
La dee da

### Verse 5

4 r 7 6 4 r 4 3
And I was running far away

4 r 7 6 7 9 7 7
Would I run off the world someday?

7 7 9 6
Nobody knows

6 7 6 9 0 7 6 r 4
Nobody knows, and

r 7 6 4 r 7 6
I was dancing in the rain

4 r 7 6 6 7 9 7 u 7
I felt alive and I can't complain

7 [ 9 u 7
But now take me home

9 0 7 9 u 7 6 6 7
Take me home where I belong

6 7 6 7 6 7 u u 7
I got no other place to go

[ 9 u 7
Now take me home

9 0 7 9 u 7 9 9 7
Take me home where I belong

6 7 6 7 6 7 u u 7
I got no other place to go

[ 9 u 7
Now take me home

9 0 7 9 u 7 6 7
Take me home where I belong

9 0 7 9 u 7 6 7 u 7 u 7
I can't take it anymore

### Verse 6

9 9 0 [ 0
But I kept running

9 9 [ 9 0 0 9 7
For a soft place to fall

9 7 9 9 7
And I kept running

9 7 9 6 6 6
For a soft place to fall

9 9 0 [ 0 9
And I kept running

9 9 [ 9 0 0 9 0 7
For a soft place to fall

9 7 9 9 7
And I kept running

9 7 9 6 6 6 7 9
For a soft place to fall

### Verse 7

4 r 7 6 4 r 4 3
And I was running far away

4 r 7 6 7 9 7 7
Would I run off the world someday?

7 [ 9 u 7
But now take me home

9 0 7 u 7 6 6 6 7
Take me home where I belong

6 7 6 7 6 7 u u 7
I got no other place to go

[ 9 u 7
Now take me home

9 0 7 9 u 7 9 9 7
Take me home where I belong

6 7 6 7 6 7 u u 7
I got no other place to go

9 9 0 [ -
Now take me home

0 - [ - 0 0 9
Home where I belong

9 7 9 7

9 9 0 [ -
Now take me home

0 - [ - 9 [ 9
Home where I belong

[ 9 [ 9

9 9 0 [ -
Now take me home

0 - [ - 0 0 9
Home where I belong

9 7 9 u 7 6 7 u 7 u 7
I can't take it anymore
---------------------------------------------
Another one
qwq
I can't help it okey it's what I do in my free time since I love the piano and stuff

**Comment by MiguelFitz** [comment]
Im almost  in verse 5, the only thing is that in the first verse i dont remember how to make not showing lyrics, so i just put "..."
and also i havent separete them into verse, so i think that means you´ll have to play it kind of "Dark soul mode", one mistake and you´ll have to start over, i´ll try to fix it, this is my first time in this, so please be pacient with me :D
